pkg/capabilities/consensus/ocr3/transmitter_test.go

```diff
@@ -0,0 +1,160 @@
+package ocr3
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jonboulle/clockwork"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	pbtypes "github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/mocks"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+func TestTransmitter(t *testing.T) {
+	ctx := tests.Context(t)
+	lggr := logger.Test(t)
+	s := newStore()
+
+	weid := uuid.New().String()
+
+	cp := newCapability(
+		s,
+		clockwork.NewFakeClock(),
+		10*time.Second,
+		func(config *values.Map) (pbtypes.Encoder, error) { return &encoder{}, nil },
+		lggr,
+		10,
+	)
+	cp.Start(ctx)
+	defer cp.Close()
+
+	payload, err := values.NewMap(map[string]any{"observations": []string{"something happened"}})
+	require.NoError(t, err)
+	gotCh, err := cp.Execute(ctx, capabilities.CapabilityRequest{
+		Metadata: capabilities.RequestMetadata{
+			WorkflowExecutionID: weid,
+			WorkflowID:          workflowTestID,
```

Comment:
were is workflowTestID defined? why isn't it local to this test? is there some hidden state somewhere?

Reply:
Yeah it's a test-level const :) I've refactored it to have it as a test-local variable.